src/library/trackset/crate/crate.h

```diff
@@ -1,20 +1,17 @@
-#ifndef MIXXX_CRATE_H
-#define MIXXX_CRATE_H
-
-
-#include "library/crate/crateid.h"
+#pragma once
 
+#include "library/trackset/crate/crateid.h"
 #include "util/db/dbnamedentity.h"
 
-
-class Crate: public DbNamedEntity<CrateId> {
-public:
+class Crate : public DbNamedEntity<CrateId> {
+  public:
     explicit Crate(CrateId id = CrateId())
-        : DbNamedEntity(id),
-          m_locked(false),
-          m_autoDjSource(false) {
+            : DbNamedEntity(id),
+              m_locked(false),
+              m_autoDjSource(false) {
+    }
+    ~Crate() override {
```

Comment:
Please replace the empty implementation with `= default` when touching those lines.

Reply:
They weren't touched in this PR, note my opening comment. But I'll use that in the future.

Reply:
Fixed in #2715 with https://github.com/mixxxdj/mixxx/pull/2715/commits/d98243811e3564ffe32ccf157797a5245eea4042

Reply:
I cannot review multiple PRs in parallel. I am not a human diff machine.